README.md

```diff
@@ -1,189 +1,785 @@
+6. Ensure your Blockstack compiles successfully.
+
+
+
+## Create the hello-android project
+
+In this section, you'll create an Android application in Android Studio.  You'll
+run the application in the emulator to test it.
+
+### Create a simple project
+
+In this section, you create an inital project. You'll validate the application's
+iniatial state by creating an emulator to run it in. Open Android Studio and do the following:
+
+
+1. Open Android Studio and choose **Start a new Andriod Studio project**.
+
+   If studio is already started, choose **File > New > New Project**.
+
+2. Enter these fields in the **Create Android Project** page.
+
+    <table>
+    <tr>
+      <th>Application Name</th>
+      <td><code>hello-android</code></td>
+    </tr>
+    <tr>
+      <th>Company domain</th>
+      <td><code><i>USERNAME</i>.example.com</code></td>
+    </tr>
+    <tr>
+      <th>Project location</td>
+      <td><code>/Users/<i>USERNAME</i>/AndroidStudioProjects/helloandroid</code></td>
+    </tr>
+    <tr>
+      <th>Include Kotlin support</th>
+      <td>Set (checked)</td>
+    </tr>
+    </table>
+
+3. Press **Next** to display **Target Android Devices**.
+4. Check **Phone and Tablet**.
+5. Choose API 27: Andriod 8.1 (Oreo) for the target version.
+6. Press **Next**.
+7. Choose **Empty Activity** and press **Next**.
+8. Leave the **Configure Activity** dialog with its defaults.
+
+   ![](images/configure-activity.png)
+
+9. Press **Finish**.
+
+   Android studio builds your initial project. This can take a bit the first time you do it.
+
+   ![](images/initial-build.png)
+
+### Run the app in an emulator
+
+In this section, you run the appliation and create an emulator when prompted.
+
+1. Once the project is imported into studio, click the `app` module in the **Project** window.
+
+2. Then, select **Run > Run** (or click the green arrow  in the toolbar).
+
+   Studio prompts you to **Select Deployment Target**.
+
+3. Choose **Create New Virtual Device** and press **OK**.
+
+    Studio prompts you to **Select Hardware**.
+
+4. Choose a Phone running Pixel XL.
+
+    ![](images/select-hdw.png)
+
+    Studio prompts you for a system image.
+
+5. Choose **Oreo** which is API level 27 and press **Next**.
+
+    ![](images/oreo-api.png)
+
+    Studio asks you to verify your new emulator configuration.
+
+6. Press **Finish**.
+
+    The emulation takes a moment to build. Then, studio launches the emulation and opens your application.
+
+    ![](images/hello-andriod-1.png)
+
+
+### Configure your application with the Blockstack SDK
+
+Now that you have created your initial project and verified it running in an emulator, you are ready to begin configuring the application for use with Blockstack.
+
+1. In studio, open the `AndroidManifest.xml` file.
+2. Add an `<intent-filter>` with the custom handler for Blockstack.
+
+    ```XML
+    <intent-filter>
+      <action android:name="android.intent.action.VIEW" />
```

Comment:
the schema element is missing with the custom schema

Reply:
Our final example code was simplified.  So, I'm matching what @larrysalibra 's last code was here.